services/order/main_test.go

```
@@ -0,0 +1,114 @@
+		InventoryServiceURL:      "http://localhost:8084",
+		RecommendationServiceURL: "http://localhost:8085",
+	}
+
+	serverImpl, err := NewOrderServer(cfg)
+	if err != nil {
+		t.Fatalf("failed to create OrderServer: %v", err)
+	}
+
+	handler := serverImpl.Routes()
+
+	req := httptest.NewRequest("GET", "/healthz", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)